doc/development.md

```diff
@@ -28,24 +28,23 @@ Install additional packages for Fedora, CentOS, and RHEL:
 
     sudo dnf install -y `cat automation/check-patch.packages`
 
-Create virtual environment for vdsm:
-
-    make venv
-
 
 ## Building Vdsm
 
-To configure sources (run `./configure --help` to see configuration options):
+To prepare sources (create Makefiles and configure script):
 
-    git clean -xfd
     ./autogen.sh --system --enable-timestamp
```

Comment:
Do You mean the "git clean" call? (And if so: didn't we just verify that it is unnecessary to do that?) Otherwise I don't understand the comment, because I didn't change the autogen command.

Reply:
No, the `.autogen.sh ...` line.